### ISSUE TITLE
title and text search don't work (only metadata projects?)

### ISSUE THREAD

**Issue by saracarl** [benwbrum/fromthepage#3797]
The title search and text search don't work for metadata only projects.  Text should search the created metadata fields.  Title/metadata should search the title and imported metadata (which is theoretically does, but it broken here for some reason.)

> The title and text search boxes on the [overview page](https://fromthepage.com/ohiouniversitylibraries/nikolais-louis-dance) don't appear to work.

**Comment by benwbrum** [comment]
I don't think we should provide "text search" as an option on metadata-only projects.  In addition to being very hard to implement, we need to preserve the distinction between a text and its metadata.

Title/metadata search should search the title and imported metadata, but also the user-created metadata.

**Comment by benwbrum** [comment]
I've just tested this, and it looks like title/metadata search is working correctly, since it does show hits for user-created metadata.

**Comment by benwbrum** [comment]
This appears to be either already done or obviated by @worleydl's search work.  No further action needed.
